Playwright.Axe/AxeCoreWrapper/DefaultAxeCoreWrapper.cs

```diff
@@ -77,6 +80,22 @@ private static async Task<TResult> EvaluateAxeRun<TResult>(IPage page, AxeRunCon
             return DeserializeResult<TResult>(jsonObject);
         }
 
+        private async Task<AxeResults> EvaluateAxeRunPartial(IPage page, AxeRunContext? context = null, object? param = null)
+        {
+            string runPartialContent = m_contentContentProvider.GetLibContent();
+
+            string? paramString = param is null 
+                ? "{}"
+                : JsonSerializer.Serialize(param, s_jsonOptions);
+            string? contextParam = context is null 
+                ? "document" 
+                : ($"JSON.parse(\'{JsonSerializer.Serialize(context, s_jsonOptions)}\'),");
```

Comment:
This isn't a new issue from this PR (it affects `EvaluateAxeRun`, too), but I think this is going to be problematic if `context` happens to use a selector that involves an apostrophe, eg `button[aria-label="Let's start!"]`.

I'd suggest passing the variable inputs via the `arg` parameter of `Page.EvaluateAsync`, instead of embedding them in the script expression and trying to escape them manually. Something along these lines, maybe:

```csharp
string serializedContext = context is null ? null : JsonSerializer.Serialize(context, s_jsonOptions);
string serializedParam = param is null ? "{}" : JsonSerializer.Serialize(param, s_jsonOptions);

object partialRunList = await page.EvaluateAsync<object>(
    $"async ([context, param]) => await (() => {{{runPartialContent} return exposedModules.runPartialAndMerge(context === null ? document : JSON.parse(context), param, window)}})()",
    new object[] { serializedContext, serializedParam }
);
```